package.json

```
@@ -7,7 +7,7 @@
     "doc": "docs"
   },
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "pre-push-lint": "cd eq-author && yarn lint && cd .. && cd eq-author-api && yarn lint && cd .. && cd eq-publisher && yarn lint"
```

This could be shortened to: 

```suggestion
    "pre-push-lint": "cd eq-author && yarn lint && cd ../eq-author-api && yarn lint && cd ../eq-publisher && yarn lint"
```